src/syscalls/socket/mod.rs

```diff
@@ -448,6 +456,14 @@ impl From<SocketAddrV6> for sockaddr_in6 {
 	}
 }
 
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
+pub struct sockaddr_un {
+	pub sun_len: u8,
+	pub sun_family: sa_family_t,
+	pub sun_path: [c_char; 108],
```

Comment:
Let's stick to BSD as we do with the other sockaddr structs.

```suggestion
	pub sun_path: [c_char; 104],
```